CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# v1.8.0 (2022-06-08)
+
+## OS Changes
+
+### General
+* Update admin and control containers ([#2191])
+* Update to containerd 1.6.x ([#2158])
+* Restart container runtimes when certificates store changes ([#2076])
+* Add support for providing kernel parameters via Boot Configuration ([#1980])
+* Restart long-running systemd services on exit ([#2162])
+* Ignore zero blocks on dm-verity root ([#2169])
+* Add support for static DNS mappings in `/etc/hosts` ([#2129])
+* Enable network configuration generation via `netdog` ([#2066])
+* Add support for non-`eth0` default interfaces ([#2144])
+* Add c7g instance type support to AWS variants ([#2107], thanks, @lizthegrey!)
```

Comment:
This is only relevant for Kubernetes, and it's closer to
```suggestion
* Add c7g instance data for max pods calculation in AWS variants ([#2107], thanks, @lizthegrey!)
```

Reply:
Serves me right for not looking at this more closely! I'll fix it.